src/modules/ekf2/EKF/velocity_fusion.cpp

```diff
@@ -57,13 +60,13 @@ void Ekf::updateVelocityAidSrcStatus(const uint64_t &time_us, const Vector3f &ob
 {
 	resetEstimatorAidStatus(aid_src);
 
+	const int state_index = State::vel.idx;
```

Comment:
`State::vel.idx` is already a constant so it's not worth declaring this variable out of the loop just to save 3 characters in the code. It's all optimized out by the compiler, so it makes no difference.

Reply:
true, you want me to revert it?